docs/api-inference/tasks/summarization.md

```diff
@@ -21,6 +21,7 @@ This is only a subset of the supported models. Find the model that suits you bes
 
 | Payload |  |  |
 | :--- | :--- | :--- |
+| **inputs*** | _string_ | The text to summarize. |
```

Comment:
This is missing all the parameters of `text2text-generation` no?

Reply:
Yes indeed. I've made some updates in https://github.com/huggingface/huggingface.js/pull/872 and it's better now. Parameters have been added in https://github.com/huggingface/hub-docs/pull/1398/commits/825571b3d9610278cc72359d004ddef0bd865987